tasks/git-intro.md

```diff
@@ -0,0 +1,64 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# Git and GitHub
+
+Get familiar with Git and GitHub.
+
+1.  Finish the course [How to use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
+
+    You may resort to subtitles/closed captions and
+    to auto-translated subtitles in particular if you feel
+    it would help you to master the video course better.
+
+    ![youtube-CC-guide](../img/youtube-captions.png)
+
+2.  Finish [try.github.io](https://try.github.io/levels/1/challenges/1)
+
+3.  Create repository named `kottans-backend`.
+
+4.  Create `README.md` for the repository.
+
+5.  Describe your impressions about learned materials.
+
+6.  Send a pull-request to [Kottans/mock-repo][mock-repo] proposing a change.
+    It could be anything, really. If you have found a typo in README - great!
+    Take care to give your PR a meaningful name and description.
+
+    **Please note, that your PR may not be reviewed quickly.**
+
+7.  Study Extra Materials below to improve your skills.
+    If you feel it affects your overall course performance consider
+    reverting to those later e.g. when you have all mandatory tasks completed.
+
+When you finish this task you can start the next one.
+
+## Extra Materials
+
+- [Git за 30 хвилин](https://codeguida.com/post/453)
+
+- [Git tips](http://sixrevisions.com/web-development/git-tips/) — consolidate your knowledge of Git
+
+- [Learn git branching](http://learngitbranching.js.org) — improve your understanding of branching
+
+- [Communicating using Markdown](https://lab.github.com/githubtraining/communicating-using-markdown)
+
+- [TypingClub](https://www.typingclub.com/) — improve your typing speed
+
+- [Как учиться и справляться с негативными мыслями](https://hexletguides.github.io/learning/)
+
+## Done?
+
+➡️ Go forward to [Linux, Command Line, HTTP Tools](linux-cli-http.md)
+
+⤴️ Back to [Contents](../contents.md)
+
+[icon-chat]: https://img.shields.io/badge/chat-on%20telegram-blue.svg
+[icon-mit]: https://img.shields.io/badge/license-MIT-blue.svg
+[icon-awesome]: https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg
+[license]: https://github.com/Kottans/web/blob/master/LICENSE.md
+[awesome]: https://github.com/sindresorhus/awesome
+[chat]: https://t.me/joinchat/CX8EF1JmLm9IM6J6oy2U7Q
```

Comment:
Are you sure we need the link to the FE chat here too? :)

Reply:
@suchov its magically gone :)